packages/cdktf/package.json

```
@@ -71,12 +72,14 @@
       "@typescript-eslint/no-use-before-define": 0,
       "@typescript-eslint/no-empty-interface": 0,
       "@typescript-eslint/explicit-module-boundary-types": 0,
-      "no-sequences": "error"
+      "no-sequences": "error",
+      "jsdoc/require-jsdoc": [2, { "contexts": ["ClassDeclaration"] }]
```

I'd be more explicit here. Using `0` for turning off rules feels intuitive enough, but using `2` never really felt good to me 😁
```suggestion
      "jsdoc/require-jsdoc": ["error", { "contexts": ["ClassDeclaration"] }]
```